ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -84,10 +98,23 @@ export class ImportCommand extends IronfishCommand {
 
     process.stdin.off('data', onData)
 
-    return JSONUtils.parse<AccountImport>(data)
+    const convertedData = ImportCommand.bech32ToJSON(data)
+    return JSONUtils.parse<AccountImport>(convertedData !== null ? convertedData : data)
```

Comment:
Can be shortened to
`ImportCommand.bech32ToJSON(data) ?? data`

Reply:
Also it is totally possible that this JSON decode fails as well, probably want to throw a meaningful error message if that happens